app/assets/stylesheets/blogs.scss

```diff
@@ -0,0 +1,37 @@
+.blogs {
+  .blog-header {
+    @include clearfix;
+
+    .icon {
+      float: left;
+      width: 64px;
+      height: 64px;
+    }
+
+    .title {
+      float: left;
+      margin-left: 20px;
+      line-height: 60px;
+    }
+  }
+
+  .blog-colophon {
+    margin-bottom: 10px;
+    font-size: 150%;
+    padding: 10px; // Same as $_content_padding
+    background-color: #ddd;
+    border-radius: 3px;
+
+    .title {
+      color: $black;
+      margin-bottom: 0;
+    }
+  }
+
+  .pagination > span {
+    margin: 0 5px;
+    padding: 8px;
+    border-radius: 5px;
+    border: solid 1px gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.